README.md

```diff
@@ -198,17 +223,23 @@ Contributors
 Maintainer
 -----------
 
-* **Kirk Hess**
-  * [GitHub](https://github.com/kirkhess)
+* **LD4P2 Sinopia Project Team**
+  * [GitHub](https://github.com/ld4p/)
 
 
 License
 -------
 
-Unless otherwise noted, code that is original to `bfe` is in the Public Domain.
+Unless otherwise noted, code that is original to the `Sinopia Editor` is licensed
+under the [Apache 2](https://www.apache.org/licenses/LICENSE-2.0).  
```

Comment:
uh - is this true?  Do we need to change the LICENSE.txt file?   I'm all for whatever is the correct thing - let's just be correct in all places.  Stanford license vs LD4P vs LC ... ugh.

Reply:
`package.json` references ISC and MIT licenses.